src/TestStack.White/Mappings/ControlDictionary.cs

```diff
@@ -69,6 +69,7 @@ private ControlDictionary()
             items.AddWPFPrimary(typeof(Image), ControlType.Image);
             items.AddSilverlightPrimary(typeof(Image), ControlType.Image);
             items.AddWin32Primary(typeof(Image), ControlType.Image);
+            items.AddWin32Primary(typeof(Win32Window), ControlType.Window);
```

Comment:
Listing Win32Window here apparently requires that it get a standardized constructor so that the framework can instantiate it as needed.